e2e_test.go

```diff
@@ -36,7 +36,7 @@ func TestEndToEndTrustlessGatewayDomains(t *testing.T) {
 
 	args := testcmd.Args(rainbow, "--trustless-gateway-domains", "example.org")
 	ready := testcmd.NewStdoutWatcher("IPFS Gateway listening")
-	domain := testcmd.NewStdoutWatcher("RAINBOW_TRUSTLESS_GATEWAY_DOMAINS = example.org")
+	domain := testcmd.NewStdoutWatcher("RAINBOW_TRUSTLESS_GATEWAY_DOMAINS        = example.org")
```

Comment:
What strange formatting is this?

Reply:
when config is printed on rainbow startup, the "=" sign is always in the same column (padded with spaces) – just my ocd ;)